utils/Constants.ts

```diff
@@ -130,13 +132,24 @@ export const rskMultisigTreasuryAddress = "0x18bc35c3b74b35c70cff0ec14ad62f4a8c2
 export const bdxLockAmount = to_d18(3150000);
 export const bdxLockingContractAddressRSK = "0x4292Ef0D3AfA1052605e2D706349dFe3A481cDcF";
 
-export const ETH_USDC_ADDRESS = "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48"; // usdc on eth
+type SupportedToken = {
+  [key: string]: { symbol: string; address: string; decimals: number };
+};
 
-export const EXTERNAL_USD_STABLE: { [key: string]: { symbol: string; address: string; decimals: number } } = {
+export const ETH_USDC_ADDRESS = "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48"; // USDC on eth
+export const EXTERNAL_USD_STABLE: SupportedToken = {
   mainnetFork: { symbol: "USDC", address: ETH_USDC_ADDRESS, decimals: 6 },
   rsk: { symbol: "XUSD", address: RSK_XUSD_ADDRESS, decimals: 18 }
 };
 
+export const ETH_DAI_ADDRESS = "0x6B175474E89094C44Da98b954EedeAC495271d0F"; // DAI on eth
+export const SECONDARY_EXTERNAL_USD_STABLE: SupportedToken = {
```

Comment:
As discussed, let's remove the use of `EXTERNAL_USD_STABLE` and `SECONDARY_EXTERNAL_USD_STABLE` in favor of just using the list `EXTERNAL_SUPPORTED_TOKENS`

Reply:
I have second thoughts about that, there are some places we are using one of them explicitly so we can't use a generic array... in one place we could use `EXTERNAL_**USD**_SUPPORTED_TOKENS` for defining the price as $1 but it's only a single usage and we might not want to keep this logic of hardcoding the price to $1

Reply:
I see us adding support also for DAI/USDC/USDT in the future (as multichain.org comes along), would we add a single entry for each?

I looked at where we're using specifically one of them, but it seems to me like it can/should be changed to an array.. Where do you see otherwise?

Reply:
_"I see us adding support also for DAI/USDC/USDT in the future (as multichain.org comes along), would we add a single entry for each?"_
And if we will add tokens that are not $?

_"I looked at where we're using specifically one of them, but it seems to me like it can/should be changed to an array.. Where do you see otherwise?"_

9_add_bdus_xusd_pool.ts
11_add_bdus_doc_pool.ts
getListOfSupportedLiquidityPools() in constants.ts
getUsdcFor() in liquidity-pools.ts
and basically almost all of the usages...
Actually, the only place where I can see that we can use the array it's where we set the token price to $1.00 and we might prefer not to do it this way since it's a WA anyway